ports/stm/boards/pyb_nano_v2/mpconfigboard.mk

```diff
@@ -16,9 +16,9 @@ LD_FILE = boards/STM32F411_fs.ld
 # Too big for the flash
 CIRCUITPY_AUDIOCORE = 0
 CIRCUITPY_AUDIOPWMIO = 0
-CIRCUITPY_KEYPAD = 0
-CIRCUITPY_GIFIO = 0
-CIRCUITPY_MIDI = 0
-CIRCUITPY_MSGPACK = 0
```

Comment:
did midi get turned back _on_ by this change?

Reply:
There is no `CIRCUITPY_MIDI`, or at least not anymore (I can't remember if that's what it used to be) . It is `CIRCUITPY_USB_MIDI`. I left it on for this general-purpose board and turned some other stuff off.